books/RayTracingTheRestOfYourLife.html

```diff
@@ -27,16 +27,16 @@
 a highly interested student, or are someone who is pursuing a career related to ray tracing. We will
 be diving into the math of creating a very serious ray tracer. When you are done, you should be
 well equipped to use and modify the various commercial ray tracers found in many popular industries,
-such as the movie, television, product design, and architecture.
+such as the movie, television, product design, and architecture industry.
 
-There are many many things I do not cover in this short volume. There are many ways of writing Monte
-Carlo rendering programs--I dive into only one of them. I don’t cover shadow rays (instead deciding
-to make rays more likely to go toward lights), bidirectional methods, Metropolis methods, or photon
-mapping. You'll find many of these techniques in so-called "serious ray tracers", but these
-techniques are not covered here because it is more important to cover the concepts, the math, and
-the terms of the field. I think of this book as a deep exposure that can be your first of many, and
-it will equip you with some of the concepts, math, and terms that you'll need to study all of these
-interesting techniques, and, more, of course.
+There are many many things I do not cover in this short volume. For example, there are many ways of
+writing Monte Carlo rendering programs--I dive into only one of them. I don’t cover shadow rays
+(instead deciding to make rays more likely to go toward lights), nor do I cover any of bidirectional
+methods, Metropolis methods, or photon mappings. You'll find many of these techniques in the
+so-called "serious ray tracers", but they are not covered here because it is more important to cover
+the concepts, the math, and the terms of the field. I think of this book as a deep exposure that
+should be your first of many, and it will equip you with some of the concepts, math, and terms that
+you'll need to study all of these interesting techniques, and hopefully more, of course.
```

Comment:
just a thought:
I dive into only one Monte Carlo path tracer type to introduce the concepts, math, and field terms.
I do not cover shadow rays, bidirectional methods, Metropolis sampling or photon mapping.  You'll
find such in so-called "serious ray tracers."  I think of this book as a deep exposure that should
be your first of many, and it will equip you with some of the concepts, math, and terms that you'll
need to study all of these interesting techniques, and hopefully more, of course.